app/src/main/java/com/android/shelfLife/ui/overview/AddFoodItem.kt

```diff
@@ -57,9 +65,13 @@ fun AddFoodItemScreen(
   var unitExpanded by remember { mutableStateOf(false) }
   var categoryExpanded by remember { mutableStateOf(false) }
   var locationExpanded by remember { mutableStateOf(false) }
+  var selectedImage by remember { mutableStateOf<FoodFacts?>(null) }
```

Comment:
```suggestion
  var selectedImageFoodFact by remember { mutableStateOf<FoodFacts?>(null) }
```
You could change the variable name to be more clear. It represents a selectedImage, but is a `FoodFact`! not a image url

Reply:
Gotcha, will try to change this